elastic/tests/test_unit.py

```diff
@@ -168,7 +168,10 @@ def test__get_data_creates_critical_service_alert(aggregator, instance):
         check = ESCheck('elastic', {}, instances=[instance])
 
         with pytest.raises(Exception):
-            check._get_data(url='test.com')
```

Comment:
Since we're not invoking `dd_run_check` here, could we just assert on specific exception(s)? I'd vote for that over the proposed solution.

Reply:
`test__get_data_throws_authentication_error` does that already, lines 152-160 above